src/build/help/help.xml

```diff
@@ -1087,6 +1097,16 @@
                         <example>~/.ssh/id_ed25519.pub</example>
                     </config-key>
 
+                    <config-key id="repo-sftp-strict-host-key-check" name="SFTP host key check type">
+                        <summary>SFTP host key check type.</summary>
+
+                        <text>
+                            <p>The default setting is <quote>yes</quote>. If this flag is set to <quote>yes</quote>, <backrest/> will never automatically add host keys to the ~/.ssh/known_hosts file, and refuses to connect to hosts whose host key has changed or is not found in the known hosts files. This option forces the user to manually add all new hosts. If this flag is set to <quote>accept-new</quote> then <backrest/> will automatically add new host keys to the user's known hosts file, but will not permit connections to hosts with changed host keys. If this flag is set to <quote>no</quote> or <quote>off</quote>, <backrest/> will automatically add new host keys to the user's known hosts file and allow connections to hosts with changed host keys to proceed.</p>
+                        </text>
```

Comment:
Why do no and off do the same thing? Is this to emulate something in OpenSSH?

Reply:
That is how the StrictHostKeyChecking option in OpenSSH works, yes.

Reply:
Not sure if I'm excited about having legacy cruft from OpenSSH transferred to pgBackRest.

Also @jreidthompson, usually we document these options as lists (see repo-type). Any reason not to do that here?

Reply:
Didn't really see it as legacy, just how openssh options generally work.  I'm not completely set on it being this way, but it seemed to be reasonable and to match openssh.  If you really don't like that then I suppose we can change it to whatever you feel would be more pgbackrest-y.

Reply:
I'd prefer just `no`. That's still a valid OpenSSH setting. If we accept `off` shouldn't we accept `on`? :)

Reply:
If you'd like to just accept 'no' then that's fine with me.

Reply:
OK -- @jreidthompson can you remove off as an option? I'm done in this branch for the day.

Reply:
Yes. I will remove it. Will also look at the options noted above as a list.

Reply:
Removed off as an option. Updated the options to be presented as a list. Updated comments, tests, help entries.
919fac824d11bf47c4f3704a2fad72787565bea4